packages/nextra-theme-blog/src/basic-layout.tsx

```diff
@@ -10,7 +10,7 @@ export const BasicLayout = ({ children }: { children: ReactNode }) => {
   const ref = useRef<HTMLHeadingElement>(null)
   return (
     <article
-      className="nx-container nx-prose-sm dark:nx-prose-dark md:nx-prose"
+      className="nx-container nx-prose nx-prose-sm dark:nx-prose-dark md:nx-prose"
```

Comment:
Yeah mistake is that `prose` class should be always included, we can also simplify
```suggestion
      className="nx-container nx-prose max-md:nx-prose-sm dark:nx-prose-dark"
```
In this case, this line should be updated too https://github.com/shuding/nextra/blob/6f3d3f06657cb434b7514394e2a81d4d7045f4f7/packages/nextra-theme-blog/src/styles.css#L24

could you also include a changeset?

Reply:
Thanks for the quick reply! I noticed some `max-w` changes to the page when altering `md:nx-prose` so I didn't want to touch it initially, but I'll see what I can do. I will include a changeset.